modules/nw-ovn-kubernetes-get-raft-db-status.adoc

```diff
@@ -0,0 +1,125 @@
+// Module included in the following assemblies:
+//
+// * networking/ovn_kubernetes_network_provider/ovn-kubernetes-architecture.adoc
+
+:_content-type: PROCEDURE
+[id="nw-list-querying-status-northbound-southbound-databases_{context}"]
+= Querying the status of the northbound and southbound databases
+
+The Kubernetes control plane nodes each contain an `ovnkube-master` pod which hosts containers for the OVN northbound and southbound databases which must form `Raft` consensus between its peer database.
```

Comment:
ISG: Use "that" rather than "which" in this case.

Reply:
I'm having trouble parsing this sentence. Is the consensus formed between/with/by a single peer database?

Reply:
This abstract doesn't seem like it's from a procedure to me. From the mod docs documentation:

> The procedure introduction is a short paragraph that provides an overview of the module. The introduction includes what the module will help the user do and why it will be beneficial to the user. It includes key words that relate to the module to maximize search engine optimization.